application/common/dos.py

```diff
@@ -115,15 +115,27 @@ def get_matching_dos_services(odscode: str, org_type_id: str) -> List[DoSService
         list[DoSService]: List of DoSService objects with matching first 5
         digits of odscode, taken from DoS database
     """
```

Comment:
Unrelated to this line. Please can you modify the ods code checker to check for length of 7 for dentists from NHS UK
![image](https://user-images.githubusercontent.com/62281988/162224928-d480adf7-0afb-4233-a9b1-9d5b002fa7ba.png)

Reply:
Sure